.github/ISSUE_TEMPLATE/2-feature-request.md

```diff
@@ -1,15 +1,18 @@
 ---
-name: "\U0001F680 Feature request"
+name: "\U0001F680 Feature request is not here"
```

Comment:
This is the text that shows up on the CTA button; i'm not sure we'd want to discourage clicks on it without being able to point people elsewhere?

Reply:
Not sure either. Ideally would be open the issue and it be blocked to post.
Then it could have any name. But probably there no this possibility.

Reply:
I add: "Feature request is here?"